kolibri/core/content/management/commands/importchannel.py

```diff
@@ -214,13 +214,37 @@ def progress_callback(bytes):
                         .exclude(kind=content_kinds.TOPIC)
                         .values_list("id", flat=True)
                     )
+                    admin_imported_ids = list(
+                        ContentNode.objects.filter(
+                            channel_id=channel_id, available=True, admin_imported=True
+                        )
+                        .exclude(kind=content_kinds.TOPIC)
+                        .values_list("id", flat=True)
+                    )
+                    not_admin_imported_ids = list(
+                        ContentNode.objects.filter(
+                            channel_id=channel_id, available=True, admin_imported=False
+                        )
+                        .exclude(kind=content_kinds.TOPIC)
+                        .values_list("id", flat=True)
+                    )
```

Comment:
So it's important that these are queried and resolved to lists before the import, right? If so, that's a subtlety in the code that could be missed without a code comment. Especially since the 3 querysets are nearly identical and could catch someone seeking opportunities for code consolidation